rocAL/include/loaders/image/node_fused_jpeg_crop_single_shard.h

```diff
@@ -39,7 +39,7 @@ class FusedJpegCropSingleShardNode : public Node {
     /// for example if there are 10 images in the dataset and load_batch_count is 3, the loader repeats 2 images as if there are 12 images available.
     void init(unsigned shard_id, unsigned shard_count, unsigned cpu_num_threads, const std::string &source_path, const std::string &json_path, StorageType storage_type,
               DecoderType decoder_type, bool shuffle, bool loop, size_t load_batch_count, RocalMemType mem_type, std::shared_ptr<MetaDataReader> meta_data_reader,
-              unsigned num_attempts, std::vector<float> &random_area, std::vector<float> &random_aspect_ratio);
+              unsigned num_attempts, std::vector<float> &random_area, std::vector<float> &random_aspect_ratio, RocalBatchPolicy _last_batch_policy = RocalBatchPolicy::PARTIAL, bool last_batch_padded = false);
```

Comment:
Why are we using PARTIAL as default policy, is it because it is default in torch?

Reply:
last_batch_policy (optional, default = LastBatchPolicy.FILL)

in DALI - default is FILL. Lets maintain it that way. For most of the training pipelines deafult would be FILL.

Reply:
Please remove the "_" from _last_batch_policy variable name